sendgrid/helpers/exceptions/exceptions.py

```diff
@@ -0,0 +1,3 @@
+class SendGridException(Exception):
+    def __init__(self, *args, **kwargs):
```

Comment:
Maybe can reduce these lines to just "pass" since we are not doing anything here